RoaringBitmap/build.gradle.kts

```diff
@@ -1,8 +1,47 @@
+buildscript {
+  repositories {
+    maven {
+      url = uri("https://plugins.gradle.org/m2/")
+    }
+  }
+  dependencies {
+    classpath("org.javamodularity:moduleplugin:1.8.12")
+  }
+}
+
+apply(plugin = "org.javamodularity.moduleplugin")
+
+// Unset Java 8 release applied from root project to allow modularity plugin to
+// control the class file versions.
+tasks.named<JavaCompile>("compileJava") {
+    options.release.set(null as Int?)
+}
+
+configure<org.javamodularity.moduleplugin.extensions.ModularityExtension> {
+    mixedJavaRelease(8)
+}
+
+// Unset Java 8 release applied from root project to allow modularity plugin to
+// control the class file versions.
+tasks.named<JavaCompile>("compileModuleInfoJava") {
+    options.release.set(null as Int?)
+}
+
+tasks.test {
+    extensions.configure(org.javamodularity.moduleplugin.extensions.TestModuleOptions::class) {
+        setRunOnClasspath(true)
+    }
+}
+
+tasks.compileTestJava {
+    extensions.configure(org.javamodularity.moduleplugin.extensions.CompileTestModuleOptions::class) {
+        setCompileOnClasspath(true)
+    }
```

Comment:
Add comments explaining that this avoids using modules for testing which allows the tests to run on Java 8.